polaris/polaris/integrations/javascript.py

```diff
@@ -1,21 +1,40 @@
 from typing import List, Dict, Optional
 
 
-def scripts(page_content: Optional[Dict]) -> List[str]:
+class TemplateScript:
+    def __init__(self, file_path: str = None, url: str = None, is_async: bool = False):
+        if file_path and url:
+            raise AttributeError("A script can only have one source from either a file_path or url.")
+        elif not (file_path or url):
+            raise AttributeError("Must give a source for TemplateScript.")
```

Comment:
I would explicitly state what parameters are necessary here. Also start the error messages with a lower-case letter.